docs/api.md

```diff
@@ -1,9 +1,33 @@
 # Editor.js API
 
 
-Blocks have access to the public methods provided by Editor.js API Module. Plugin and Tune Developers
+Tools have access to the public methods provided by Editor.js API Module. Plugin and Tune Developers
 can use Editor\`s API as they want.
 
+## Block API
+
+API for certain Block methods and properties. You can access it through `editor.api.block.getBlockByIndex` method or get it form `block` property of [Tool constructor](../types/tools/block-tool.d.ts) argument.
+
+`name: string` — Block's Tool name
+
+`config: ToolConfig` — Tool config passed on Editor initialization
+
+`holder: HTMLElement` — HTML Element that wraps Tool's HTML content
```

Comment:
is this an element returned by tools render method? The description is not clear.

Reply:
We used this definition earlier for the same thing, is it better?
![image](https://user-images.githubusercontent.com/23050529/79701228-034f5680-82a4-11ea-8122-a624de0cb197.png)